h5py/tests/common.py

```diff
@@ -165,3 +167,29 @@ def closed_tempfile(suffix='', text=None):
             test_file.flush()
     yield file_name
     shutil.rmtree(file_name, ignore_errors=True)
+
+
+def insubprocess(f):
+    """Runs a test in its own subprocess"""
+    @wraps(f)
+    def wrapper(request, *args, **kwargs):
+        curr_test = request.node.name
```

Comment:
It looks like this only gets the function name, unfortunately, not the module name. We could try something like `request.node.function.__module__` to get the module name. Or use just the function name with the `-k` option, which would rely on there being only one test with that name across all files.

Reply:
let's try the module trick